immunopepper/spark.py

```diff
@@ -134,7 +43,7 @@ def pq_WithRenamedCols(spark, list_paths):
 
 def process_matrix_file(spark, index_name, jct_col, jct_annot_col, rf_annot_col,
                         path_matrix, whitelist, cross_junction, filterNeojuncCoord, filterAnnotatedRF,
-                        tot_batches=None, batch_id=None):
+                        separate_back_annot=False, tot_batches=None, batch_id=None):
```

Comment:
Commented already above - `separate_back_annot` is not quite descriptive. It is also not mentioned in the below description of parameters. Maybe this could be added.